src/locales/en/messages.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  messages: {
+    "Dashboard": "Dashboard",
+    "Earn": "Earn",
+    "Buy": "Buy",
+    "Referrals": "Referrals",
+    "Ecosystem": "Ecosystem",
+    "About": "About",
+    "Settings": "Settings",
+    "HOME": "HOME",
+    "EARN": "EARN",
+    "ABOUT": "ABOUT",
+    "Home": "Home",
+    "Trade": "Trade",
+    "Connect": "Connect",
+    "Connect Wallet": "Connect Wallet",
+    "MetaMask not detected.": "MetaMask not detected.",
+    "Install MetaMask": "Install MetaMask",
+    ", and use GMX with its built-in browser": ", and use GMX with its built-in browser",
+    " to start using GMX": " to start using GMX",
+    "Coinbase Wallet not detected.": "Coinbase Wallet not detected.",
+    "Install Coinbase Wallet": "Install Coinbase Wallet"
```

Comment:
Imo better to use more "key-like" keys for mappings. otherwise eventually after updating texts for several times we will end up having weird mappings that don't make sense

e.g. imagine after a while we will update text ", and use GMX with its built-in browser" to ", and gain access to trust-less trading platform". and in a mapping file it will be:
```
", and use GMX with its built-in browser": ", and gain access to trust-less trading platform"
```
it might confuse both us and people who will translate it. i've had an experience with that approach and with product in active development in can become an issue in future

i believe approach with more technical key names fits better here. e.g.
```
"instalMetamaskButton": "Install MetaMask",
"installMetamaskButtonDescription": ", and use GMX with its built-in browser"
```

also namespaces can be used here. like:
```
"nav.home": "Home"
```

Reply:
afaik you've discussed that with @xvi10 
let's wait for him in case there are some arguments against

Reply:
didn't discuss the naming convention, I think your recommendation makes sense

Reply:
@midasdev711 re-discussed with @gdev8317, let's use the text as the key for now, i thought the syntax for the translation file would be something like this though: https://github.com/Uniswap/interface/blob/main/src/locales/de-DE.po#L100